newa/__init__.py

```diff
@@ -148,7 +148,27 @@ class Erratum(Cloneable, Serializable):
     """ An eratum """
 
     release: str
-    # builds: list[...] = ...
+    builds: list[str] = []
+
+    def fetch_details(self) -> None:
+        raise NotImplementedError
+
+
+@define
+class Issue(Cloneable, Serializable):
+    """ A Jira issue """
+
+    issue: str
```

Comment:
Maybe `id`? `issue.issue` looks like a typo, or a parent/child relationship, `issue.id` might be clearer.

Reply:
Right you are. I got confused by a conflict with `id` property but that one is defined for a different class, not this one.

Reply:
Changed.